packages/docs/src/pages/contribution/ui-element-guide.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <docs-content :config="configComputed" />
+</template>
+
+<script lang="ts">
+import { Vue, Options } from 'vue-class-component'
+import DocsContent from '../../components/DocsContent.vue'
+import uiElementGuideConfig from '../../components/page-configs/contribution/ui-element-guide/page-config'
+
+@Options({
```

Comment:
Please, add here components name, it really helps to do refactoring with dev tools, #923